docs/en/reference/types.rst

```diff
@@ -927,7 +927,10 @@ Now we implement our ``Doctrine\DBAL\Types\Type`` instance:
         }
     }
 
-The job of Doctrine-DBAL is to transform your type into SQL declaration. You can modify the SQL declaration Doctrine will produce. At first, you must to enable this feature by overriding the canRequireSQLConversion method:
+The job of Doctrine-DBAL is to transform your type into SQL declaration.
```

Comment:
And to convert values?

Reply:
Or is that covered in the rest of this article?

Reply:
It is covered just below, on line 943